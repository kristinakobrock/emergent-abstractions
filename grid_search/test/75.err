  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 26.71it/s] 14%|#############                                                                                | 7/50 [00:00<00:01, 29.92it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 29.47it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 26.20it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 25.76it/s] 38%|##################################9                                                         | 19/50 [00:00<00:01, 25.36it/s] 44%|########################################4                                                   | 22/50 [00:00<00:01, 26.53it/s] 50%|##############################################                                              | 25/50 [00:00<00:00, 26.08it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.53it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 23.06it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 21.71it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 21.86it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 21.97it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 22.14it/s] 94%|######################################################################################4     | 47/50 [00:01<00:00, 26.38it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.42it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 23%|#####################4                                                                       | 3/13 [00:00<00:00, 28.84it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 23.85it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 21.37it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 24.01it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.07it/s]
