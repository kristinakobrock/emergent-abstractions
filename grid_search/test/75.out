# ["--attributes=5", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.99", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6828708648681641, "acc": 0.6030742526054382, "length": 2.8567423820495605, "mode": "train", "epoch": 1}
{"loss": 0.6650075912475586, "acc": 0.6244140267372131, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6613545417785645, "acc": 0.6287425756454468, "length": 1.201907753944397, "mode": "train", "epoch": 2}
{"loss": 0.6641873717308044, "acc": 0.6257812976837158, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6598162651062012, "acc": 0.6290167570114136, "length": 1.1135661602020264, "mode": "train", "epoch": 3}
{"loss": 0.6629942655563354, "acc": 0.6253905892372131, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6586396098136902, "acc": 0.6279503703117371, "length": 1.3078879117965698, "mode": "train", "epoch": 4}
{"loss": 0.6658070087432861, "acc": 0.6220703125, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.643825888633728, "acc": 0.6365131139755249, "length": 2.094759702682495, "mode": "train", "epoch": 5}
{"loss": 0.6462664008140564, "acc": 0.6298889517784119, "length": 1.8203125, "mode": "test", "epoch": 5}
{"loss": 0.6250498294830322, "acc": 0.6599956750869751, "length": 2.172811508178711, "mode": "train", "epoch": 6}
{"loss": 0.6207020878791809, "acc": 0.6584351062774658, "length": 2.1015625, "mode": "test", "epoch": 6}
{"loss": 0.60104900598526, "acc": 0.6790750622749329, "length": 2.3934834003448486, "mode": "train", "epoch": 7}
{"loss": 0.5980623364448547, "acc": 0.6806458234786987, "length": 2.62890625, "mode": "test", "epoch": 7}
{"loss": 0.5580407977104187, "acc": 0.7169691920280457, "length": 2.8463149070739746, "mode": "train", "epoch": 8}
{"loss": 0.5854411721229553, "acc": 0.6973938345909119, "length": 2.9453125, "mode": "test", "epoch": 8}
{"loss": 0.5333831310272217, "acc": 0.736122727394104, "length": 2.8946805000305176, "mode": "train", "epoch": 9}
{"loss": 0.5685737729072571, "acc": 0.7063964605331421, "length": 3.17578125, "mode": "test", "epoch": 9}
{"loss": 0.516916036605835, "acc": 0.7455024719238281, "length": 3.031466007232666, "mode": "train", "epoch": 10}
{"loss": 0.5546128749847412, "acc": 0.7109862565994263, "length": 3.33984375, "mode": "test", "epoch": 10}
{"loss": 0.5008793473243713, "acc": 0.7574976086616516, "length": 3.1809630393981934, "mode": "train", "epoch": 11}
{"loss": 0.5463330149650574, "acc": 0.724395751953125, "length": 3.2265625, "mode": "test", "epoch": 11}
{"loss": 0.48278865218162537, "acc": 0.7686734199523926, "length": 3.2602484226226807, "mode": "train", "epoch": 12}
{"loss": 0.5709172487258911, "acc": 0.7232605814933777, "length": 2.953125, "mode": "test", "epoch": 12}
{"loss": 0.47082754969596863, "acc": 0.7751477956771851, "length": 3.3509607315063477, "mode": "train", "epoch": 13}
{"loss": 0.5488792657852173, "acc": 0.7404846549034119, "length": 3.26953125, "mode": "test", "epoch": 13}
{"loss": 0.4562552869319916, "acc": 0.7839189767837524, "length": 3.3918092250823975, "mode": "train", "epoch": 14}
{"loss": 0.5255136489868164, "acc": 0.76080322265625, "length": 3.46875, "mode": "test", "epoch": 14}
{"loss": 0.4405651092529297, "acc": 0.7917235493659973, "length": 3.4852447509765625, "mode": "train", "epoch": 15}
{"loss": 0.5052619576454163, "acc": 0.7653014659881592, "length": 3.60546875, "mode": "test", "epoch": 15}
{"loss": 0.43209174275398254, "acc": 0.7995536923408508, "length": 3.5616724491119385, "mode": "train", "epoch": 16}
{"loss": 0.4813165068626404, "acc": 0.7758179306983948, "length": 3.54296875, "mode": "test", "epoch": 16}
{"loss": 0.4210304319858551, "acc": 0.8041256070137024, "length": 3.652132272720337, "mode": "train", "epoch": 17}
{"loss": 0.48548227548599243, "acc": 0.7761168479919434, "length": 3.640625, "mode": "test", "epoch": 17}
{"loss": 0.41186317801475525, "acc": 0.8106200098991394, "length": 3.7154340744018555, "mode": "train", "epoch": 18}
{"loss": 0.48109644651412964, "acc": 0.7822875380516052, "length": 3.6640625, "mode": "test", "epoch": 18}
{"loss": 0.40645721554756165, "acc": 0.8123356103897095, "length": 3.7059357166290283, "mode": "train", "epoch": 19}
{"loss": 0.4690733551979065, "acc": 0.7801880240440369, "length": 3.72265625, "mode": "test", "epoch": 19}
{"loss": 0.39958980679512024, "acc": 0.8163688778877258, "length": 3.75897479057312, "mode": "train", "epoch": 20}
{"loss": 0.4597298502922058, "acc": 0.785980224609375, "length": 3.74609375, "mode": "test", "epoch": 20}
{"loss": 0.38728371262550354, "acc": 0.8209304213523865, "length": 3.7858259677886963, "mode": "train", "epoch": 21}
{"loss": 0.4612809717655182, "acc": 0.790985107421875, "length": 3.76953125, "mode": "test", "epoch": 21}
{"loss": 0.38441815972328186, "acc": 0.8245749473571777, "length": 3.8277976512908936, "mode": "train", "epoch": 22}
{"loss": 0.45595645904541016, "acc": 0.785064697265625, "length": 3.94140625, "mode": "test", "epoch": 22}
{"loss": 0.3804739713668823, "acc": 0.8204198479652405, "length": 3.8803629875183105, "mode": "train", "epoch": 23}
{"loss": 0.457450807094574, "acc": 0.786651611328125, "length": 3.95703125, "mode": "test", "epoch": 23}
{"loss": 0.37719491124153137, "acc": 0.8230735659599304, "length": 3.8834285736083984, "mode": "train", "epoch": 24}
{"loss": 0.45597410202026367, "acc": 0.7890991568565369, "length": 3.984375, "mode": "test", "epoch": 24}
{"loss": 0.36634641885757446, "acc": 0.8311681747436523, "length": 3.918954849243164, "mode": "train", "epoch": 25}
{"loss": 0.4429451525211334, "acc": 0.7947266101837158, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.36509907245635986, "acc": 0.8309127688407898, "length": 3.956336736679077, "mode": "train", "epoch": 26}
{"loss": 0.4512742757797241, "acc": 0.7904296517372131, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.35939809679985046, "acc": 0.8303921818733215, "length": 3.958336114883423, "mode": "train", "epoch": 27}
{"loss": 0.4592941999435425, "acc": 0.7808594107627869, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.35222527384757996, "acc": 0.8380320072174072, "length": 3.973010778427124, "mode": "train", "epoch": 28}
{"loss": 0.45639920234680176, "acc": 0.7736328840255737, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.344923734664917, "acc": 0.8398526906967163, "length": 3.9804294109344482, "mode": "train", "epoch": 29}
{"loss": 0.45589479804039, "acc": 0.7906250357627869, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.33966338634490967, "acc": 0.8377129435539246, "length": 3.985058546066284, "mode": "train", "epoch": 30}
{"loss": 0.4679000675678253, "acc": 0.771484375, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3377527594566345, "acc": 0.8438081741333008, "length": 3.9871485233306885, "mode": "train", "epoch": 31}
{"loss": 0.45900920033454895, "acc": 0.7878905534744263, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.33358344435691833, "acc": 0.8493059277534485, "length": 3.9913277626037598, "mode": "train", "epoch": 32}
{"loss": 0.4579642713069916, "acc": 0.7894530892372131, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3228584825992584, "acc": 0.852106511592865, "length": 3.993917942047119, "mode": "train", "epoch": 33}
{"loss": 0.46972528100013733, "acc": 0.7865234017372131, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.31900152564048767, "acc": 0.8551385402679443, "length": 3.9931111335754395, "mode": "train", "epoch": 34}
{"loss": 0.47302156686782837, "acc": 0.7871093153953552, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3185120224952698, "acc": 0.8554834723472595, "length": 3.991777181625366, "mode": "train", "epoch": 35}
{"loss": 0.4589623212814331, "acc": 0.794921875, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.31427180767059326, "acc": 0.858956515789032, "length": 3.9950473308563232, "mode": "train", "epoch": 36}
{"loss": 0.4669019281864166, "acc": 0.7919921875, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3109723627567291, "acc": 0.8602229356765747, "length": 3.9942786693573, "mode": "train", "epoch": 37}
{"loss": 0.4524039626121521, "acc": 0.7908202409744263, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3074280023574829, "acc": 0.8650294542312622, "length": 3.9940025806427, "mode": "train", "epoch": 38}
{"loss": 0.48257747292518616, "acc": 0.7822265625, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.303962379693985, "acc": 0.8655648231506348, "length": 3.9972543716430664, "mode": "train", "epoch": 39}
{"loss": 0.44686809182167053, "acc": 0.80078125, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2941626310348511, "acc": 0.8707623481750488, "length": 3.997347116470337, "mode": "train", "epoch": 40}
{"loss": 0.4363607168197632, "acc": 0.8042969107627869, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.28682807087898254, "acc": 0.8764593601226807, "length": 3.997922897338867, "mode": "train", "epoch": 41}
{"loss": 0.4637640416622162, "acc": 0.8007811903953552, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.2791602909564972, "acc": 0.8804343342781067, "length": 3.9974052906036377, "mode": "train", "epoch": 42}
{"loss": 0.4792264401912689, "acc": 0.80078125, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2781318724155426, "acc": 0.8811399936676025, "length": 3.997817039489746, "mode": "train", "epoch": 43}
{"loss": 0.44603919982910156, "acc": 0.8121094107627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.27305296063423157, "acc": 0.8830464482307434, "length": 3.9969301223754883, "mode": "train", "epoch": 44}
{"loss": 0.45360150933265686, "acc": 0.8113280534744263, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.26983389258384705, "acc": 0.8859302997589111, "length": 3.9974632263183594, "mode": "train", "epoch": 45}
{"loss": 0.47392064332962036, "acc": 0.8054686188697815, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.25999653339385986, "acc": 0.8894115090370178, "length": 3.998269557952881, "mode": "train", "epoch": 46}
{"loss": 0.45022648572921753, "acc": 0.8082031607627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.2539229094982147, "acc": 0.8940699696540833, "length": 3.9979145526885986, "mode": "train", "epoch": 47}
{"loss": 0.43674448132514954, "acc": 0.8220702409744263, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.24535472691059113, "acc": 0.8994845747947693, "length": 3.9987306594848633, "mode": "train", "epoch": 48}
{"loss": 0.4333304464817047, "acc": 0.8259764909744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.24142950773239136, "acc": 0.9018763303756714, "length": 3.9982550144195557, "mode": "train", "epoch": 49}
{"loss": 0.41398516297340393, "acc": 0.8357421159744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2383585274219513, "acc": 0.901712954044342, "length": 3.9988033771514893, "mode": "train", "epoch": 50}
{"loss": 0.42436376214027405, "acc": 0.8330078125, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2384510636329651, "acc": 0.9015264511108398, "length": 3.9978830814361572, "mode": "train", "epoch": 51}
{"loss": 0.39550158381462097, "acc": 0.8515624403953552, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2282092273235321, "acc": 0.9090335369110107, "length": 3.996852159500122, "mode": "train", "epoch": 52}
{"loss": 0.42206448316574097, "acc": 0.8392577767372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.22112567722797394, "acc": 0.9121899008750916, "length": 3.998551845550537, "mode": "train", "epoch": 53}
{"loss": 0.42475244402885437, "acc": 0.8312500715255737, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.222063347697258, "acc": 0.9094259738922119, "length": 3.998684883117676, "mode": "train", "epoch": 54}
{"loss": 0.3988165855407715, "acc": 0.8468751311302185, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.223519966006279, "acc": 0.9105532169342041, "length": 3.9991374015808105, "mode": "train", "epoch": 55}
{"loss": 0.4077688753604889, "acc": 0.8460937142372131, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.21456556022167206, "acc": 0.9130154848098755, "length": 3.9991512298583984, "mode": "train", "epoch": 56}
{"loss": 0.4147801697254181, "acc": 0.8333984017372131, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.20991428196430206, "acc": 0.9166054725646973, "length": 3.999410629272461, "mode": "train", "epoch": 57}
{"loss": 0.4190925657749176, "acc": 0.8382812142372131, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.21233072876930237, "acc": 0.9158440828323364, "length": 3.9994845390319824, "mode": "train", "epoch": 58}
{"loss": 0.41685938835144043, "acc": 0.8525390028953552, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.20216992497444153, "acc": 0.920407772064209, "length": 3.9993503093719482, "mode": "train", "epoch": 59}
{"loss": 0.42109888792037964, "acc": 0.8363280892372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.1978880614042282, "acc": 0.9206563234329224, "length": 3.9977447986602783, "mode": "train", "epoch": 60}
{"loss": 0.42007046937942505, "acc": 0.8468750715255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7563333511352539
